src/main/java/pl/sda/weather/repository/impl/LocationRepositoryImpl.java

```diff
@@ -0,0 +1,92 @@
+package pl.sda.weather.repository.impl;
+
+
+import com.google.gson.Gson;
+import pl.sda.weather.repository.ILocationRepository;
+import pl.sda.weather.model.LocationModel;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LocationRepositoryImpl implements ILocationRepository {
+
+    private static final String FILE_LOCATION = "src/main/resources/location/locations.txt";
```

Comment:
would be nice to have test for that also -> this FILE_LOCATION also should be move from constant to constructor